.github/workflows/setup-cloud-sdk.yml

```diff
@@ -1,37 +1,51 @@
-name: setup-cloud-sdk Unit
+name: 'unit'
 
-on: 
+on:
   push:
     branches:
-      - main
+    - 'main'
   pull_request:
+    branches:
+    - 'main'
+  workflow_dispatch:
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
 
 jobs:
   run:
-    name: Unit Tests
-    if: ${{ github.event.pull_request.head.user.login != 'google-github-actions-bot'}}
-    runs-on: ${{ matrix.operating-system }}
+    name: 'unit'
+    runs-on: '${{ matrix.os }}'
     strategy:
       matrix:
-        operating-system: [ubuntu-latest, windows-latest, macos-latest]
-        node: [10, 12, 14]
+        os:
+        - 'ubuntu-latest'
+        - 'windows-latest'
+        - 'macos-latest'
+        node:
+        - '10'
+        - '12'
+        - '14'
     steps:
-    - uses: actions/checkout@v2
+    - uses: 'actions/checkout@v2'
 
-    - uses: actions/setup-node@master
+    - uses: 'actions/setup-node@v2'
       with:
-        node-version: 12.x
+        node-version: '${{ matrix.node }}.x'
```

Comment:
I think this was a CI bug - we were never actually setting the node.js version (we were running node 12 for all tests)